code/game/objects/items/trader.dm

```diff
@@ -1018,3 +1018,150 @@ var/global/list/alcatraz_stuff = list(
 					var/mob/living/living_mover = mover
 					living_mover.put_in_hands(I)
 				to_chat(mover,"<span class='good'>\The [src] dispenses a reward!</span>")
+
+
+//Mystery mob cubes//////////////
+
+/obj/item/weapon/storage/box/mysterycubes
+	name = "mystery cube box"
+	desc = "Dehydrated friends!"
+	icon = 'icons/obj/pbag.dmi'
+	icon_state = "pbag"	//Supposed to look kind of shitty, cubes aren't even wrapped
+	foldable = /obj/item/weapon/paper
+	can_only_hold = list("/obj/item/weapon/reagent_containers/food/snacks/monkeycube/mysterycube")
+
+/obj/item/weapon/storage/box/mysterycubes/New()
+	..()
+	var/friendAmount = 1
+	friendAmount = rand(1, 3)
+	for(var/i = 1 to friendAmount)
+		new /obj/item/weapon/reagent_containers/food/snacks/monkeycube/mysterycube(src)
+
+/obj/item/weapon/reagent_containers/food/snacks/monkeycube/mysterycube
+	name = "mystery cube"
+	desc = "A portable friend!"
+	var/list/potentialFriends = list()
+
+/obj/item/weapon/reagent_containers/food/snacks/monkeycube/mysterycube/New()
+	..()
+	potentialFriends = existing_typesof(/mob/living/simple_animal) - (boss_mobs + blacklisted_mobs)
+	contained_mob = pick(potentialFriends)
+
+
+//Mystery chem beakers//////////////
+
+/obj/item/weapon/storage/box/mystery_vial
+	name = "assorted chemical pack"
+	desc = "A mix of reagents from who knows where."
+	icon_state = "beaker"
+
+/obj/item/weapon/storage/box/mystery_vial/New()
+	..()
+	for(var/i = 1 to 5)
+		new /obj/item/weapon/reagent_containers/glass/beaker/vial/mystery(src)
+
+/obj/item/weapon/reagent_containers/glass/beaker/vial/mystery
+	name = "recycled vial"
+	desc = "Slightly scratched and worn, it looks like this wasn't its original purpose. The label has been sloppily peeled off."
+	mech_flags = MECH_SCAN_FAIL	//Nip that in the bud
+	var/list/illegalChems = list(	//Just a bad idea
+		ADMINORDRAZINE,
+		BLOCKIZINE,
+		AUTISTNANITES,
+		XENOMICROBES,
+		PAISMOKE
```

Comment:
`PAISMOKE` is fun and also not that hard to obtain legitimately if you're very clever and/or were told how
Of course, it's up to you if you still want it to be blacklisted. Just putting it out there.

Reply:
Is it possible to obtain in a beaker? I'll likely keep it blacklisted just for safety though, and because it shouldn't really be its own chem in universe.

Reply:
It is indeed possible to get it into a beaker. It's not hard if you know how, but it is hard to figure out how.

Reply:
I once witnessed a guy get PaiSmoke. I haven't the faintest idea how he did it, but it's possible.